### PR TITLE
Add basic communication things

### PR DESCRIPTION
**Pull request by uduse**

### CODE REVIEW

File: gobanana/comm/task.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import uuid
+
+from gobanana import comm
+
+
+class Task(comm.DataTransferObject):
+    @classmethod
```
Comment (7FeiW) [inline gobanana/comm/task.py]: do you mean @staticmethod? since getid() does not need any class variables

File: gobanana/comm/comm.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+from gobanana import comm
+
+
+def get_client():
+    return redis.Redis()
+
+
+class TaskDispatcher(object):
+    def __init__(self, task: 'comm.Task'):
+        self.task = task
```
Comment (7FeiW) [inline gobanana/comm/comm.py]: Are we going to use _foo for private member? Does have to do that,  I just want it to be consistent.

File: gobanana/comm/task.py (hunk 1 above)
Comment (uduse) [reply gobanana/comm/task.py]: It's a common practice to only use `classmethod` and ignore `staticmethod`.

File: gobanana/comm/comm.py (hunk 2 above)
Comment (uduse) [reply gobanana/comm/comm.py]: Yes, using underscores for privates is usually a better practice. However, I think it's fine to ignore that for classes that are too simple like this. Maybe I will refactor this class later.

**Pull request merged**